install-dev/upgrade/sql/1.6.0.11.sql

```diff
@@ -24,8 +24,6 @@ ALTER TABLE `PREFIX_product_attribute_shop` MODIFY `unit_price_impact` DECIMAL(2
 
 /* PHP:p16011_media_server(); */;
 
-INSERT INTO `PREFIX_configuration` (`name`, `value`, `date_add`, `date_upd`) VALUES ('PS_PRICE_DISPLAY_PRECISION', 2, NOW(), NOW());
```

Comment:
Do not remove upgrade script :thinking: 
Add the 
```sql
DELETE `PREFIX_configuration` WHERE name = 'PS_PRICE_DISPLAY_PRECISION';
```

in the 1.7.7.0.sql instead

Reply:
oh ok, I thought that update scripts where all played successively at install, my mistake.

Reply:
done !